csrc/ir/utils.cpp

```diff
@@ -1099,6 +1099,31 @@ int64_t getVectorizeSize(const TensorView* tv) {
   return 1;
 }
 
+bool hasTrivialAllocationDomain(const TensorView* tv) {
```

Comment:
The intention of this utility is to generalize `!tv->hasAllocation()` to cases where an allocation domain is provided, but it actually corresponds to the no-reductions rfactor domain (ignoring broadcasts).

Reply:
Thanks! Would it be possible to add this as a comment in the header file.